testnets/cometbft_config_template.toml

```
@@ -290,10 +290,11 @@ wal_dir = ""
 # Maximum number of transactions in the mempool
 size = 5000
 
-# Limit the total size of all txs in the mempool.
-# This only accounts for raw transactions (e.g. given 1MB transactions and
-# max_txs_bytes=5MB, mempool will only accept 5 transactions).
-max_txs_bytes = 1073741824
+# The maximum size in bytes of all transactions stored in the mempool.
+# This is the raw, total transaction size. For example, given 1MB
+# transactions and a 5MB maximum mempool byte size, the mempool will
+# only accept five transactions.
+max_txs_bytes = 67108864
```

```suggestion
max_txs_bytes = 5242880
```

About x5 the block size, this is ultimately a user controlled parameter

---

For posterity's sake: the block size is being set to 1MiB in #4597.